packages/components/src/tools-panel/test/index.js

```diff
@@ -425,4 +432,36 @@ describe( 'ToolsPanel', () => {
 			expect( altMenuItem ).toHaveAttribute( 'aria-checked', 'false' );
 		} );
 	} );
+
+	describe( 'panel header icon toggle', () => {
+		it( 'should continue to render shown by default item after it is toggled off via menu item', async () => {
```

Comment:
I think we have a copy/paste error here.
```suggestion
		it( 'should render appropriate icons for the dropdown menu', async () => {
```

Reply:
Thanks for spotting that.